### ISSUE TITLE
[Request] Expose Lambda memory as variables

### ISSUE THREAD

**Issue by skyzyx** [philips-labs/terraform-aws-github-runner#1314]
terraform-aws-github-runner v0.21.0 \
Terraform 1.0.9

While setting this up and having it not work, I started digging through logs and discovered that the Lambda functions were being killed for OOM. Looking at your Terraform, you're only specifying a memory value for one of the Lambdas (256 MB), but leaving the others undefined — this appears to use 128 MB as a default value.

This request is to expose these memory options as variables at the top level, so that I/we don’t have to fork and customize.

I'm opening this as a request instead of a PR because of its overall simplicity, but if this isn't a high priority for the core team I could find the brain-space to wire-up the Terraform and open a PR myself in the near-term.

Thanks!

**Comment by npalm** [comment]
@mcaulifn maybe we can simply add this now you working on the PR #1228

**Comment by skyzyx** [comment]
Due to the slow response from this group _historically_, as well as the tepid response to my PRs, I ended up hard-forking this project. I've stripped out the things that we didn't need, streamlined the pipeline, and started rewriting some bits in Go to get better performance and faster cold-start times.

I have no opinion on whether or not my PRs and Issues are responded to because I'm now using a fork that is better tuned for our use-cases.
